universal-application-tool-0.0.1/app/repository/StoredFileRepository.java

```diff
@@ -31,6 +31,18 @@ public StoredFileRepository(
     return supplyAsync(() -> ebeanServer.find(StoredFile.class).findSet(), executionContext);
   }
 
+  public CompletionStage<Set<StoredFile>> listWithPresignedURL() {
```

Comment:
nit: as per the style guide:
```suggestion
  public CompletionStage<Set<StoredFile>> listWithPresignedUrl() {
```

Reply:
Done.